galaxy.yml

```diff
@@ -1,10 +1,8 @@
----
-
 ### REQUIRED
 # The namespace of the collection. This can be a company/brand/organization or product namespace under which all
 # content lives. May only contain alphanumeric lowercase characters and underscores. Namespaces cannot start with
 # underscores or numbers and cannot contain consecutive underscores
-namespace: community
+namespace: sap
```

Comment:
namespace should be community

Reply:
That is misunderstanding. 
Collection that has been developed and in this PR is called sap.sap_operations.
We do not intend to develop collection in community namespace as this is not eligible to be published on AAP.

Reply:
Kirill, here the tool from Bernd comes into place. When you push to AutomationHub it automatically changes the namespace to sap and only picks the supported part.

Reply:
Does it work with python code? 
How it handles ansible dependencies? 
And then users from AAP click on the link they see collection community.sap_operations, but they are looking for sap.sap_operations. How any tool will help with user experience?